exercises/concept/intro-select/test-result.jq

```diff
@@ -0,0 +1,30 @@
+def columns:
+    if (. | length) == 0 then []
+    else .[0] | keys
+    end;
+
+def rows:
+    map(to_entries | map(.value));
+
+def failure_message(got; expected):
+    (got | columns | tostring)  as $got_columns
+    | (expected | columns | tostring) as $expected_columns
+    | if $got_columns != $expected_columns then
+        "Expected columns " + $expected_columns + "; but got " + $got_columns
+      else
+        (got | rows | tostring) as $got_rows
+        | (expected | rows | tostring) as $expected_rows
+        | "With columns " + $got_columns + ", \nbexpected " + $expected_rows + ";\nbut got " + $got_rows
```

Comment:
little typo
```suggestion
        | "With columns " + $got_columns + ", \nexpected " + $expected_rows + ";\nbut got " + $got_rows
```

Reply:
Shouldn't this be a string interpolation?
```suggestion
        | "With columns \($got_columns)\nexpected \($expected_rows\)\nbut got \($got_rows)"
```